docs/releases/1.8.txt

```diff
@@ -559,6 +559,10 @@ Requests and Responses
   <django.http.HttpResponse.setdefault>` method allows setting a header unless
   it has already been set.
 
+* The new :class:`~django.http.FileResponse` subclass of
+  :class:`~django.http.StreamingHttpResponse` tries to efficiently stream
```

Comment:
This "tries to" language is a bit vague. "You can use the new FileResponse class to efficiently stream files."